src/Components/CookieConsent.js

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import * as Scrivito from "scrivito";
+import ReactCookieConsent from "react-cookie-consent";
+import cookieConsentIcon from "../assets/images/cookie_consent_icon.svg";
+
+function CookieConsent() {
+  const root = Scrivito.Obj.root();
+  const cookieLink = root.get("cookieConsentLink");
```

Comment:
If `root` is `null` this will fail.

For details see https://www.scrivito.com/obj-root-f3e15c4a58f46c68

Reply:
Also one should usually try to name one thing with as few names as possible. Therefore I suggest this:

```js
const cookieConsentLink = root.get("cookieConsentLink");
```